api/Navigator.json

```
@@ -1832,15 +1832,19 @@
               "version_added": "13",
               "notes": [
                 "Allowed schemes include <code>mailto</code>, <code>mms</code>, <code>nntp</code>, <code>rtsp</code>, and <code>webcal</code>. Custom protocols must be prefixed with <code>web+</code>.",
-                "From Chrome 77, the URL parameter only accepts <code>http</code> or <code>https</code> URLs."
+                "From Chrome 77, the URL parameter only accepts <code>http</code> or <code>https</code> URLs.",
+                "The third <code>title</code> argument has not been removed and so will still need to be specified when using this method"
```

suggestion: revise the note to make it a bit more direct about what Web developers need to do with this information

```suggestion
                "The obsolete <code>title</code> argument is required."
```